packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -7,8 +7,10 @@ const MenuLinks = ({ is_logged_in, items }) => (
     <React.Fragment>
         {!!items.length && (
             <div className='header__menu-links'>
-                {items.map((item, idx) =>
-                    item.login_only && item.login_only !== is_logged_in ? null : (
+                {items.map((item, idx) => {
+                    const item_text = item.text();
```

Comment:
the change helps as fix the console error visible in dev environment:
```
Failed prop type: MenuLinks: prop type `items[0].text` is invalid; it must be a function, usually from the `prop-types` package, but received `undefined`.
    in MenuLinks (created by DefaultHeader)
    in DefaultHeader (created by Context.Consumer)......
```
 + it helps agains one more error saying that the title and the child passed to <Text> must be a string